### DISCUSSION TITLE
Can I use atoms to save instances of custom Classes

### DISCUSSION THREAD

**Discussion by zhangchi0104** [pmndrs/jotai#2780] [Q&A] [answered]
Hi, 

I am currently using jotai with twilio-react-native. Currently, twilio has a class called `Call` with has methods attached to the object

Given that I understand the issue of referential equality and there's no need to serialize it, is that possible for me to save it in the jotai store?

Thanks

**Comment by dai-shi** [comment, marked as answer]
Yes, you can store anything. (one exception is a bare function, because it's confused with an updating function. We can wrap the function.)
